packages/analytics/metric-cards/src/components/cardUtils.ts

```diff
@@ -0,0 +1,50 @@
+const DECIMAL_DISPLAY = 2
```

Comment:
FWIW you should move this file out of the components folder; either something like `src/utilities.ts` or a composable in `src/composables/`